src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -320,24 +338,28 @@ uiModules.get('apps/management')
 
     let lastPromise;
     resetIndex();
-    samplePromise = lastPromise = updateSamples()
+    samplePromise = lastPromise = Promise.resolve()
```

Comment:
Happy to, do you think it would help it I made it look more like this? https://github.com/elastic/kibana/pull/12007/files#diff-f5a949c353fd77eda951919524c3b7ecR199